packages/dd-trace/src/dogstatsd.js

```diff
@@ -193,6 +194,117 @@ class DogStatsDClient {
   }
 }
 
+// TODO: Handle arrays of tags and tags translation.
+class MetricsAggregationClient {
```

Comment:
```suggestion
class MetricsAggregationClient {
  _counters = {}
  _gauges = {}
  _histograms = {}
```

Reply:
This is already covered in the constructor through `reset()`